package.json

```diff
@@ -49,16 +49,17 @@
     "@types/nunjucks": "^3.1.1",
     "@types/redis": "^2.8.10",
     "@types/supertest": "^2.0.7",
-    "jest": "^24.1.0",
-    "mockserver": "^3.0.0",
+    "jest": "^24.9.0",
+    "mockserver": "^3.1.1",
     "node-mocks-http": "^1.7.6",
     "node-sass": "^4.14.1",
     "nodemon": "^1.18.9",
     "sonarqube-scanner": "^2.6.0",
     "supertest": "^3.4.2",
-    "ts-jest": "^24.1.0",
+    "ts-jest": "^25.5.1",
     "ts-node": "^8.0.2",
     "tslint": "^5.13.0",
-    "typescript": "^3.3.1"
+    "typescript": "^3.3.1",
+    "yargs-parser": "^18.1.3"
```

Comment:
you've got a new package here, did this get added from npm audit fix?

Reply:
yargs-parser

Reply:
No I added it as a fix to one of the vulnerabilities I was getting

Reply:
Ah nevermind, running an audit without it produces the same result, I'll revert it.

Reply:
Yes you won't need it.